terraform/workspaces/infra/s3/s3-logs.tf

```diff
@@ -28,19 +28,42 @@ data "aws_iam_policy_document" "logs_bucket_policy" {
   count = var.disable_logs ? 0 : 1
 
   statement {
-    sid     = "AllowPutObjects"
+    sid     = "AllowAccessLogs"
     actions = ["s3:PutObject"]
     effect  = "Allow"
     resources = [
       "${aws_s3_bucket.logs[count.index].arn}",
       "${aws_s3_bucket.logs[count.index].arn}/access_logs/AWSLogs/${data.aws_caller_identity.current.account_id}/*",
     ]
-
     principals {
       type        = "AWS"
       identifiers = [data.aws_elb_service_account.main.arn]
     }
   }
+
+  statement {
+    sid = "AllowAppLogs"
```

Comment:
It seems that we're also using this bucket for ingress logs:
https://github.com/useparagon/aws-on-prem/blob/v2.16.0/terraform/workspaces/paragon/helm/helm.tf#L238

Would the storage of ingress logs mess up how OpenObserve parses files?

Reply:
No. This is why I bumped the move to Glacier from 7 to 30 days though. It will still work after 30 days but there would be an extra change if we queried that data.